opentelemetry-tracing/opentelemetry-go/gorilla-mux/server/main.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+
+	"github.com/gorilla/mux"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
+	"github.com/signalfx/splunk-otel-go/distro"
+	"github.com/signalfx/splunk-otel-go/instrumentation/net/http/splunkhttp"
+)
+
+func main() {
+	exitCode := 0
+	defer func() {
+		os.Exit(exitCode)
+	}()
+
+	// handle CTRL+C gracefully
+	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
+	defer stop()
+
+	// initialize Splunk OTel distro
+	sdk, err := distro.Run()
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err := sdk.Shutdown(context.Background()); err != nil {
+			log.Println(err)
+			exitCode = 1
+		}
+	}()
+
+	r := mux.NewRouter()
+
+	// instrument http.Handler
+	r.Use(otelmux.Middleware("mux-server"))
+	rumHandler := splunkhttp.NewHandler(r)
+	otelHandler := otelhttp.NewHandler(rumHandler, "http-server")
```

Comment:
Won't the `otelhttp` instrumentation duplicate the `mux` instrumentation spans?

Reply:
It will duplicate a lot. The only addition is the `http.wrote_bytes` attribute. I will remove `otelhttp` from here.

Reply:
Addressed https://github.com/signalfx/tracing-examples/pull/106/commits/6f9adb0a155ddcb4a37b57f1a1bcd18859f40144